packages/flutter/lib/src/widgets/modal_barrier.dart

```diff
@@ -214,6 +340,24 @@ class AnimatedModalBarrier extends AnimatedWidget {
   /// {@macro flutter.widgets.ModalBarrier.onDismiss}
   final VoidCallback? onDismiss;
 
+  /// This [ValueNotifier] contains a value of type [EdgeInsets] that specifies
+  /// how the [SemanticsNode.rect] of the barrier should be clipped (so that
+  /// it does not overlap with the content rendered on top of it).
+  ///
+  /// See also"
+  ///
+  ///  * [SemanticsClipper], which utilizes the value inside this [ValueNotifier]
+  /// to update the [SemanticsNode.rect] for its child.
+  final ValueNotifier<EdgeInsets>? clipDetailsNotifier;
+
+  /// This hint text instructs user what they are able to do when they tap on
+  /// the [ModalBarrier], annouced in the format of 'Double tap to
+  /// $[semanticsOnTapHint].'
+  ///
+  /// If this value is null, the default onTapHint will be applied, resulting
+  /// in the annoucement of 'Double tap to activate'.
+  final String? semanticsOnTapHint;
```

Comment:
I think these are the same as above. Please define macros for them and use them here to avoid duplicating documentation

Reply:
Thanks! Done.